sdk/Cargo.toml

```diff
@@ -72,7 +72,6 @@ num-derive = { workspace = true }
 num-traits = { workspace = true }
 num_enum = { workspace = true }
 pbkdf2 = { workspace = true }
-qstring = { workspace = true }
```

Comment:
I can't comment further up, but can you also remove `derivation-path` as a dependency?